xm-commons-function/src/main/java/com/icthh/xm/commons/config/FunctionApiSpecConfiguration.java

```diff
@@ -0,0 +1,70 @@
+package com.icthh.xm.commons.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.icthh.xm.commons.config.client.api.RefreshableConfiguration;
+import com.icthh.xm.commons.domain.spec.FunctionApiSpec;
+import com.icthh.xm.commons.logging.aop.IgnoreLogginAspect;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.AntPathMatcher;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Slf4j
+@Service
+@IgnoreLogginAspect
+public class FunctionApiSpecConfiguration implements RefreshableConfiguration { // todo: do we need MapRefreshableConfiguration (???)
```

Comment:
can we resolve this todo?

Reply:
are you sure that it's necessary to implement MapRefreshableConfiguration here? i'm not sure that other files would be involved into configuration soon and this change could be done when it's really necessary. @amedvedchuk would you share any thoughts about this, please?

Reply:
Realized that `MapRefreshableConfiguration` will be useful for input data spec processing (second step). My suggestion is to make this changes in the next pr related to this task